doc/formatter_simplified_class_diagram.graphml

```diff
@@ -0,0 +1,180 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Maybe we should document how to regenerate the PNG from from the graphml file.

So personally, I'd probably just suggest using a Makefile right in this directory as BUILD feels like too big of a stone for that bird....

Reply:
I found a few markdown enhancements that support more inline rendering:
https://github.com/shd101wyy/markdown-preview-enhanced/blob/master/docs/diagrams.md
Could be worth a look.

I'm also wondering if the open-source codesearch already provides visualization tools.  They have full semantic information to be able to navigate class hierarchies (kythe).  Having the code as source-of-truth again mitigates risk of documentation going stale.

Reply:
Wow, that is an interesting list of things that can be pre-viewed in github.

I was always annoyed that they don't allow viewing SVGs (because it allows potentially dangerous scripting), but this certainly can help in many of the documentation situations.

Reply:
mmh, I tried it,  looks like this is not supported by github. These previews there are done by an add-on for a [particular editor](atom.io) (and that was a very bad 'particle pun').